OrbitGl/AsyncTrack.h

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_GL_ASYNC_TRACK_H_
+#define ORBIT_GL_ASYNC_TRACK_H_
+
+#include <string>
+
+#include "ThreadTrack.h"
```

Comment:
I think this should be TimerTrack.h (class below is derived from TimerTrack).

Reply:
Fixed, thanks.